sdk/test-utils/test-credential/review/test-credential.api.md

```diff
@@ -5,25 +5,17 @@
 ```ts
 
 import { AccessToken } from '@azure/core-auth';
-import { ClientSecretCredential } from '@azure/identity';
-import { TokenCredential } from '@azure/core-auth';
-import { TokenCredentialOptions } from '@azure/identity';
+import { DefaultAzureCredentialClientIdOptions } from '@azure/identity';
+import { DefaultAzureCredentialOptions } from '@azure/identity';
+import { DefaultAzureCredentialResourceIdOptions } from '@azure/identity';
+import { TokenCredential } from '@azure/identity';
+import { TokenCredential as TokenCredential_2 } from '@azure/core-auth';
```

Comment:
Quick question, what's the difference between the `TokenCredential` from the two package? Can we update a better name than `TokenCredential_2`?